Examples/restcomm-helloworld/app/build.gradle

```diff
@@ -47,7 +47,7 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-   // compile project(':restcomm.android.sdk')
+    //compile project(':restcomm.android.sdk')
     compile 'org.restcomm:restcomm-android-sdk:1.0.5-BETA7-174'
```

Comment:
Since our maven artifact is already [out](https://search.maven.org/#artifactdetails%7Corg.restcomm%7Crestcomm-android-sdk%7C1.0.6-BETA8-303%7Caar), let's update the compile dependency to it and verify it works with Hello World as discussed. If all is good, commit the whole thing